x-pack/plugins/secops/public/store/local/network/actions.ts

```diff
@@ -32,9 +34,10 @@ export const updateIsPtrIncluded = actionCreator<{
   networkType: NetworkType;
 }>('UPDATE_DNS_iS_PTR_INCLUDED');
```

Comment:
Uh oh .... shouldn't that whole piece of text be a capital SCREAM. Someone forgot an `i` at some point.

Reply:
Fixed! 🍨